### ISSUE TITLE
why are the y ones in get_data_generator for gan_trainer?

### ISSUE THREAD

**Issue by peterlee909** [leafinity/keras_ganomaly#3]
```
def get_data_generator(data, batch_size=32):
    datalen = len(data)
    cnt = 0
    while True:
        idxes = np.arange(datalen)
        np.random.shuffle(idxes)
        cnt += 1
        for i in range(int(np.ceil(datalen/batch_size))):
            train_x = np.take(data, idxes[i*batch_size: (i+1) * batch_size], axis=0)
            y = np.ones(len(train_x))
            yield train_x, [y, y, y]
```

Why are the y ones instead of zeros? The loss have to close to zero, right?
Do I misunderstand something?

**Comment by leafinity** [comment]
Ganomaly doesn't use label  in loss function, so it can be assigned to be any value.
Why I generate y value is that when calling train_on_batch, y value is needed.
This program is the sample code of the article below, you can find the explanation of each part there.
https://medium.com/@radleaf/draw-a-similar-image-to-do-anomaly-detection-228b7cd188e3
